commandline/src/main/java/tech/pegasys/web3signer/commandline/convertor/YamlMapperCliConverter.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2023 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.web3signer.commandline.convertor;
+
+import static tech.pegasys.web3signer.commandline.Web3SignerBaseCommand.KEY_STORE_CONFIG_FILE_SIZE_OPTION_NAME;
+
+import tech.pegasys.web3signer.signing.config.metadata.parser.YamlMapperFactory;
+
+import com.fasterxml.jackson.dataformat.yaml.YAMLMapper;
+import picocli.CommandLine;
+import picocli.CommandLine.TypeConversionException;
+
+public class YamlMapperCliConverter implements CommandLine.ITypeConverter<YAMLMapper> {
```

Comment:
Don't really this approach. This is using picocli convertor to initialise part of our application. Would prefer that picocli was just used for the configuration. Transforming a --key-store-config-file-max-size to a YamlMapper doesn't make sense to me. It's a value that is used as part of creating it not converting to another type.

Having said that don't dislike it enough to block merging this PR.